src/remediations/controller.write.js

```diff
@@ -170,7 +170,24 @@ exports.patch = errors.async(async function (req, res) {
         }
 
         if (name) {
+            const existing_remediation = await db.remediation.findOne({
+                attributes: ['id'],
+                where: { created_by: username, name: name },
```

Comment:
Isn't this missing a tenant condition?

Reply:
account / org check

Reply:
Ah yes, an org id check would make sense here. I'll add that.